### PR TITLE
Show updating day and time on PM page

### PR DESCRIPTION
**Pull request by 70ray**
implements task 1719

### CODE REVIEW

**Comment by 70ray** [comment]
Although the localisation works for currently implemented DP locales we should have a more generic procedure which cannot break in the future, i.e. try to load a file that does not exist.

**Comment by 70ray** [comment]
Do we want to link to external sites or put the js files on our site?
In either case perhaps there should be some documented procedure for updating to the latest versions.

File: scripts/time.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/*global $ setInterval moment serverTimezone*/
```
Comment (cpeel) [inline scripts/time.js]: As-is this code is intimately tied to the PM page so I would prefer it to be next to the page that uses it (`tools/project_manager/projectmgr.js`).

File: tools/project_manager/projectmgr.php [hunk 2]
```diff
@@ -50,8 +50,25 @@
...
+$server_timezone = date_default_timezone_get();
+$lang_code = short_lang_code();
+$js_files = [
+    "$code_url/tools/dropdown.js",
+    "$code_url/scripts/time.js",
+    "https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.24.0/moment.js",
+    "https://cdnjs.cloudflare.com/ajax/libs/moment-timezone/0.5.23/moment-timezone-with-data.js"
+];
+
+if($lang_code != "en")
```
Comment (cpeel) [inline tools/project_manager/projectmgr.php]: I assume(?) that if there isn't a locale file available that moment will fall back to English, which is great. That at least helps buffer the case where a DP language will be added that moment doesn't support.

File: scripts/time.js (hunk 1 above)
Comment (70ray) [reply scripts/time.js]: Done this.

**Pull request merged**